scripts/switch-colors.sh

```diff
@@ -24,10 +24,10 @@ fi
 UUID=$(aws lambda list-event-source-mappings --function-name "arn:aws:lambda:us-east-1:${AWS_ACCOUNT_ID}:function:streams_${ENV}_${DEPLOYING_COLOR}" --region us-east-1 | jq -r ".EventSourceMappings[0].UUID")
 aws lambda update-event-source-mapping --uuid "${UUID}" --region us-east-1
 
-node ./web-client/switch-public-ui-colors.js
-node ./web-client/switch-ui-colors.js
-node ./web-client/switch-api-colors.js
-node ./web-client/switch-public-api-colors.js
+node ../web-client/switch-public-ui-colors.js
+node ../web-client/switch-ui-colors.js
+node ../web-client/switch-api-colors.js
+node ../web-client/switch-public-api-colors.js
```

Comment:
Curious about this one. It does prevent manual switching of colors. Why are the leading periods required?

Reply:
Pushed a fix. Turns out the switch-colors had not been working at all since this refactor, but it wasn't failing on the error during deploys. I added a line to make sure it fails if there's an error. Thank you for the catch!